### PR TITLE
BZ1941060: adding explicit admin login command

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1941060

4.7+

Preview: https://deploy-preview-34127--osdocs.netlify.app/openshift-enterprise/latest/backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.html

### CODE REVIEW

**Comment by geliu2016** [comment]
@kalexand-rh LGTM, and I just think about there is other way I often used for cluster login: export KUBECONFIG=/dir/kubeconfig, anyway, it's enough to provide a login way. thanks

**Comment by ahardin-rh** [comment]
LGTM!

File: modules/dr-restoring-cluster-state.adoc [hunk 1]
```diff
@@ -185,6 +185,14 @@ etcd-ip-10-0-143-125.ec2.internal                1/1     Running     1
 +
 If the status is `Pending`, or the output lists more than one running etcd pod, wait a few minutes and check again.
 
+. In a separate terminal window, log in to the cluster as a user with the `cluster-admin` role by using the following command:
++
+[source,terminal]
+----
+# oc login -u <cluster-admin> <1>
```
Comment (bergerhoffer) [inline modules/dr-restoring-cluster-state.adoc:192]: @kalexand-rh In looking at these docs today, I noticed that this command was the only one in the procedure that used `#` as the prompt. Was that purposeful, or could this be a `$`?

Comment (kalexand-rh) [reply modules/dr-restoring-cluster-state.adoc:192]: The command provided in the bug was run as root, and I don't see any other specific requirements for it.

Comment (bergerhoffer) [reply modules/dr-restoring-cluster-state.adoc:192]: Thanks, I'll submit a PR to adjust it and will run it past QE again

**Pull request merged**